rdr_service/api/genomic_cloud_tasks_api.py

```diff
@@ -263,6 +263,48 @@ def post(self):
         return {"success": True}
 
 
+class IngestCVLManifestTaskApi(BaseGenomicTaskApi):
+    """
+    Cloud Task endpoint: Ingest CVL Manifest.
+    """
+    def post(self):
+        super(IngestCVLManifestTaskApi, self).post()
+
+        cvl_map = {
```

Comment:
Nit: can we name this something like `cvl_manifest_map` to be more specific? The name `cvl_map` makes me think it contains the actual CVL names, like Color, Baylor, UW.

Reply:
Sounds good.